1-js/02-first-steps/14-switch/2-rewrite-if-switch/task.md

```diff
@@ -1,10 +1,10 @@
-importance: 4
+importância: 4
```

Comment:
**O nome da chave `inportance` no começo dos arquivos `task.md` deve ser mantido.**

Não traduzir as chaves de identificação dos metadados, pois não fazem parte do conteúdo do artigo, mas sim para fins do projeto ["server"](https://github.com/javascript-tutorial/server) que construirá o site "https://javascript.info/", sendo assim, o conteúdo acima de --- provavelmente é utilizado pela plataforma para alguma operação ou estruturação necessária.

Reply:
That is true. I was doing it before, and need to correct where done. Thanks.